src/nav/nav.spec.ts

```diff
@@ -861,6 +874,140 @@ describe('nav', () => {
     fixture.detectChanges();
     expectContents(fixture, ['1-false', '2-true'], 1);
   });
+
+  describe(`nav keyboard navigation`, () => {
```

Comment:
Code coverage went down with this PR, here is what's not covered (you can see it in `/coverage` after running `yarn test`):

![Screen Shot 2020-03-19 at 13 30 16](https://user-images.githubusercontent.com/8074436/77067859-eb0ac400-69e5-11ea-9652-6f85d5406dc9.png)

So, basically at least:

- [ ] vertical keyboard navigation
- [ ] it would also be nice to have at least one test with disabled navs